src/software/simulated_tests/simulated_play_test_fixture_test.cpp

```diff
@@ -264,3 +285,35 @@ TEST_F(SimulatedPlayTestFixtureTest,
     runTest(terminating_validation_functions, non_terminating_validation_functions,
             Duration::fromSeconds(0.5));
```

Comment:
The test "test_terminating_and_non_terminating_validation_functions_pass_together" still uses all gtest EXPECT statements, should they be using yield?

Reply:
good points